fiftyone/core/dataset.py

```diff
@@ -314,6 +315,7 @@ def __init__(
         overwrite=False,
         _create=True,
         _virtual=False,
+        _reload=False,
```

Comment:
noticed this when resolving conflict... is this used/needed?

Reply:
Yes. It's a little tricky but while it is not used in this class' `__init__` function, it is passed to `fiftyone.core.DatasetSingleton.__call__()` where it is used if the dataset comes from the singleton cache, and unused if the dataset is new or if singleton cache is disabled.